core/renderers/zelos/info.js

```diff
@@ -188,41 +183,41 @@ RenderInfo.prototype.getInRowSpacing_ = function(prev, next) {
 /**
  * @override
  */
-RenderInfo.prototype.getSpacerRowHeight_ = function(
-    prev, next) {
+RenderInfo.prototype.getSpacerRowHeight_ = function(prev, next) {
   // If we have an empty block add a spacer to increase the height.
-  if (Types.isTopRow(prev) &&
-      Types.isBottomRow(next)) {
+  if (Types.isTopRow(prev) && Types.isBottomRow(next)) {
     return this.constants_.EMPTY_BLOCK_SPACER_HEIGHT;
   }
-  const followsStatement =
-      Types.isInputRow(prev) && prev.hasStatement;
-  const precedesStatement =
-      Types.isInputRow(next) && next.hasStatement;
+  const followsStatement = Types.isInputRow(prev) && prev.hasStatement;
+  const precedesStatement = Types.isInputRow(next) && next.hasStatement;
   if (precedesStatement || followsStatement) {
     const cornerHeight = this.constants_.INSIDE_CORNERS.rightHeight || 0;
     const height = Math.max(this.constants_.NOTCH_HEIGHT, cornerHeight);
     return precedesStatement && followsStatement ?
-        Math.max(height, this.constants_.DUMMY_INPUT_MIN_HEIGHT) : height;
+        Math.max(height, this.constants_.DUMMY_INPUT_MIN_HEIGHT) :
+        height;
   }
   // Top and bottom rows act as a spacer so we don't need any extra padding.
   if ((Types.isTopRow(prev))) {
     if (!prev.hasPreviousConnection &&
         (!this.outputConnection || this.hasStatementInput)) {
-      return Math.abs(this.constants_.NOTCH_HEIGHT -
-          this.constants_.CORNER_RADIUS);
+      return Math.abs(
+          this.constants_.NOTCH_HEIGHT - this.constants_.CORNER_RADIUS);
     }
     return this.constants_.NO_PADDING;
   }
   if ((Types.isBottomRow(next))) {
     if (!this.outputConnection) {
-      const topHeight = Math.max(this.topRow.minHeight,
-          Math.max(this.constants_.NOTCH_HEIGHT,
-              this.constants_.CORNER_RADIUS)) - this.constants_.CORNER_RADIUS;
+      const topHeight = Math.max(
+                            this.topRow.minHeight,
+                            Math.max(
+                                this.constants_.NOTCH_HEIGHT,
+                                this.constants_.CORNER_RADIUS)) -
+          this.constants_.CORNER_RADIUS;
```

Comment:
Personally I don't like the horizontal alignment, but up to you on this one.

Reply:
This is what clang-format updates it to. I don't think it's worth fighting.